ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh

```diff
@@ -54,14 +54,24 @@ timeout -s 9 175m ssh $SSHOPTS root@$IP bash - << EOF |& sed -e 's/.*auths.*/***
 
 set -ex
 
-yum install -y git
+yum install -y git sysstat sos
+systemctl start sysstat
 
 mkdir -p /tmp/artifacts
 
 mkdir dev-scripts
 tar -xzvf dev-scripts.tar.gz -C /root/dev-scripts
 chown -R root:root dev-scripts
 
+NVME_DEVICE="/dev/nvme0n1"
+if [ -e "\$NVME_DEVICE" ];
+then
+  echo 'type=83' | sudo sfdisk "\$NVME_DEVICE"
+  mkfs.xfs "\${NVME_DEVICE}p1"
```

Comment:
From CI -
Error accessing specified device /dev/nvme0n1p1: No such file or directory

you should be able to just use /dev/nvme0n1 directly if you want, without needing to partition it

Reply:
Also use "mkfs.xfs -f" as the disk may have had a FS on it.